src/main/webapp/index.html

```diff
@@ -110,14 +110,24 @@ <h1 class="text-center">Home</h1>
           </div>
         </div>
         <div class="col-md-1">
-          <button type="submit" class="btn btn-outline-success my-2 my-sm-0" onclick="searchReceipts()">
+          <button type="submit" class="btn btn-outline-success my-2 my-sm-0" onclick="getMatchingReceipts()">
             Search
           </button>
         </div>
       </div>
     </form>
     <!-- The columns will automatically stack on top of each other when the screen is less than 768px wide. -->
     <div id="receipts-display" class="row"></div>
+    <div class="row">
+      <div class="col-6 d-flex justify-content-end pb-4">
+        <button type="button" class="btn btn-info" onclick="scrollToTop(); getPreviousPageOfReceipts()">
+          &lt; Back
+        </button>
+      </div>
+      <div class="col-6 d-flex justify-content-start pb-4">
+        <button type="button" class="btn btn-info" onclick="scrollToTop(); getNextPageOfReceipts()">Next &gt;</button>
```

Comment:
I think it would be better to move the `scrollToTop()` call inside the `getPreviousPageOfReceipts()` and `getNextPageOfReceipts()` methods so that less implementation details are in the HTML.